docs/infra/destroy-infrastructure.md

```diff
@@ -4,14 +4,14 @@ To destroy everything you'll need to undeploy all the infrastructure in reverse
 
 ## Instructions
 
-1. First destroy all your environments by running `terraform destroy` in each of your environment module folders
+1. First destroy all your environments. Within `/infra/app/service` run the following, replacing `dev` with the environment you're destroying. 
 
     ```bash
-    # within each env folder in /infra/envs/ (dev, stage, etc)
-    terraform destroy
+    $ terraform init --backend-config=dev.s3.tfbackend
+    $ terraform destroy -var-file=dev.tfvars
     ```
 
-2. Then to destroy the backends, first you'll need to add `force_destroy = true` to the S3 buckets, and update the lifecycle block to set `prevent_destroy = false`. Then run `terraform apply`. The reason we need to do this is because S3 buckets by default are protected from destruction to avoid loss of data. See [Terraform: Destroy/Replace Buckets](https://medium.com/interleap/terraform-destroy-replace-buckets-cf9d63d0029d) for a more in depth explanation.
+1. Then to destroy the backends, first you'll need to add `force_destroy = true` to the S3 buckets, and update the lifecycle block to set `prevent_destroy = false`. Then run `terraform apply` from within the `infra/accounts` directory. The reason we need to do this is because S3 buckets by default are protected from destruction to avoid loss of data. See [Terraform: Destroy/Replace Buckets](https://medium.com/interleap/terraform-destroy-replace-buckets-cf9d63d0029d) for a more in depth explanation.
```

Comment:
optional nit: I have a slight preference to have markdown human readable even in text form (e.g. without a markdown renderer like github) so I like having the numbers be 1,2,3 instead of 1,1,1 (and then rely on VS code markdown extensions to autoupdate the numbers)

Reply:
Sounds good, I'll change this